plugins/clipboard/plugin.js

```diff
@@ -2539,6 +2580,227 @@
 			return undefined;
 		}
 	};
+
+	/**
+	 * Fallback dataTransfer object which is used together with {@link CKEDITOR.plugins.clipboard.dataTransfer}
+	 * for browsers supporting Clipboard API, but not supporting custom MIME types (Edge 16+ - #962).
+	 *
+	 * @since 4.8
```

Comment:
We should use semver compatible numbering in docs, so it should be `@since 4.8.0`.